[2025-08-06 17:27:22,923] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-06T15:24:19.707165+00:00 [queued]>
[2025-08-06 17:27:22,938] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-06T15:24:19.707165+00:00 [queued]>
[2025-08-06 17:27:22,938] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-06 17:27:22,938] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2025-08-06 17:27:22,938] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-06 17:27:22,957] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): fetch_and_send_weather> on 2025-08-06 15:24:19.707165+00:00
[2025-08-06 17:27:22,964] {standard_task_runner.py:52} INFO - Started process 24926 to run task
[2025-08-06 17:27:22,968] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'fetch_and_send_weather', 'scheduled__2025-08-06T15:24:19.707165+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpaytpyk_c', '--error-file', '/tmp/tmpsjbyv3qv']
[2025-08-06 17:27:22,970] {standard_task_runner.py:80} INFO - Job 38: Subtask fetch_and_send_weather
[2025-08-06 17:27:22,991] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-06 17:27:23,050] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.fetch_and_send_weather scheduled__2025-08-06T15:24:19.707165+00:00 [running]> on host 536c12e0b7f9
[2025-08-06 17:27:23,157] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=fetch_and_send_weather
AIRFLOW_CTX_EXECUTION_DATE=2025-08-06T15:24:19.707165+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-06T15:24:19.707165+00:00
[2025-08-06 17:27:23,159] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-06 17:27:23,159] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/scripts && python3 kafka/producer.py']
[2025-08-06 17:27:23,178] {subprocess.py:85} INFO - Output:
[2025-08-06 17:27:24,414] {subprocess.py:92} INFO - Envoi vers Kafka : {'temperature': 31, 'humidity': 66, 'windSpeed': 4.1, 'cloudCover': 66, 'rainIntensity': 0, 'timestamp': '2025-08-06T17:27:00Z'}
[2025-08-06 17:27:24,477] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-06 17:27:24,530] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=fetch_and_send_weather, execution_date=20250806T152419, start_date=20250806T172722, end_date=20250806T172724
[2025-08-06 17:27:24,585] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-06 17:27:24,645] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
